### PR TITLE
Added a Contributors.md

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**

### CODE REVIEW

File: Contributors.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+### [RCOS](https://rcos.io/projects/HighSchoolHacking/GLS/profile) RPI Students
+
+* [Carter Gleason](https://github.com/GleasonC4)
+    * Fall 2016:
+        * Null checking [design](https://github.com/HighSchoolHacking/GLS/pull/126]) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/135)
+        * For each [design](https://github.com/HighSchoolHacking/GLS/pull/138) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/143)
+        * [Exceptions](https://github.com/HighSchoolHacking/GLS/pulls?utf8=%E2%9C%93&q=is%3Apr%20exceptions)
+        * Varous code cleanups (dictionaries, comment blocks, files, main, while, print)
```
Comment (Ben-Wolf) [inline Contributors.md]: Typo.
"Varous"

File: Contributors.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        * JavaScript language support with Benjamin Wolf
+* [Benjamin Wolf](https://github.com/Ben-Wolf)
+    * Fall 2016:
+        * [List pops](https://github.com/HighSchoolHacking/GLS/pull/131)
```
Comment (Ben-Wolf) [inline Contributors.md]: "List Pops" seems like the odd man out, why not "List pop design and implementation"?

Comment (JoshuaKGoldberg) [inline Contributors.md]: Sure.

File: Contributors.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* [Carter Gleason](https://github.com/GleasonC4)
+    * Fall 2016:
+        * Null checking [design](https://github.com/HighSchoolHacking/GLS/pull/126]) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/135)
+        * For each [design](https://github.com/HighSchoolHacking/GLS/pull/138) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/143)
+        * [Exceptions](https://github.com/HighSchoolHacking/GLS/pulls?utf8=%E2%9C%93&q=is%3Apr%20exceptions)
+        * Various code cleanups (dictionaries, comment blocks, files, main, while, print)
+        * JavaScript language support with Benjamin Wolf
+* [Benjamin Wolf](https://github.com/Ben-Wolf)
+    * Fall 2016:
+        * [List pops design and implementation](https://github.com/HighSchoolHacking/GLS/pull/131)
```
Comment (Ben-Wolf) [inline Contributors.md]: List Pop is still a bit odd, might I suggest. 
`List pops [design](https://github.com/HighSchoolHacking/GLS/pull/128) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/131)`

Comment (JoshuaKGoldberg) [inline Contributors.md]: Interesting, for some reason I thought there was only one. Will do.

File: Contributors.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    * Fall 2016:
+        * Null checking [design](https://github.com/HighSchoolHacking/GLS/pull/126]) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/135)
+        * For each [design](https://github.com/HighSchoolHacking/GLS/pull/138) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/143)
+        * [Exceptions](https://github.com/HighSchoolHacking/GLS/pulls?utf8=%E2%9C%93&q=is%3Apr%20exceptions)
+        * Various code cleanups (dictionaries, comment blocks, files, main, while, print)
+        * JavaScript language support with Benjamin Wolf
+* [Benjamin Wolf](https://github.com/Ben-Wolf)
+    * Fall 2016:
+        * List pops [design](https://github.com/HighSchoolHacking/GLS/pull/128) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/131)
+        * [do start] and [do while](https://github.com/HighSchoolHacking/GLS/pull/139) investigations
```
Comment (JoshuaKGoldberg) [inline Contributors.md]: `[do start]`: will add link

**Comment by JoshuaKGoldberg** [comment]
👐

**Pull request merged**